modules/gdscript/editor/gdscript_highlighter.cpp

```diff
@@ -165,6 +166,9 @@ Dictionary GDScriptSyntaxHighlighter::_get_line_syntax_highlighting_impl(int p_l
 					if (in_node_path && (color_regions[in_region].start_key == "\"" || color_regions[in_region].start_key == "\'")) {
 						region_color = node_path_color;
 					}
+					if (in_string_name && (color_regions[in_region].start_key == "\"" || color_regions[in_region].start_key == "\'")) {
+						region_color = annotation_color;
```

Comment:
I'd give it its own configurable `string_name_color`.
Maybe @Calinou has a suggestion for which color to use?

Reply:
I'd also prefer if StringName had its own unique color.

As for NodePath, it is *not* the same thing as `$Node` or `%Node`, so I would prefer giving it its own color too.